bot/cogs/timeleft.py

```diff
@@ -0,0 +1,23 @@
+from discord.ext import commands
+from dateutil.relativedelta import relativedelta
+import logging
+import aiohttp
+import discord
+import datetime
+
+bot = commands.Bot(command_prefix='.', description='pr bot')
+
+@bot.command()
+async def timeleft():
+    tl = datetime.datetime(2018, 10, 31) - datetime.datetime.now()
+    days = tl.days
+    sec = tl.seconds
+    hours =  sec //3600

```

Comment:
You've mixed `datetime` methods and math here. `datetime` [provides all of these as instance attributes](https://docs.python.org/3/library/datetime.html#datetime.datetime.year) so the math is unnecessary.